src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/BaseClaimPdfTest.java

```diff
@@ -33,21 +33,21 @@ public void shouldBeAbleToFindTestClaimDataInSealedClaimPdf() throws IOException
     protected void shouldBeAbleToFindTestClaimDataInPdf(String pdfName) throws IOException {
         Claim createdCase = createCase();
         String pdfAsText = textContentOf(retrievePdf(pdfName, createdCase.getExternalId()));
-        assertionsOnClaimPdf(createdCase, pdfAsText);
+        assertionsOnPdf(createdCase, pdfAsText);
     }
 
-    private Claim createCase() {
+    protected Claim createCase() {
         ClaimData claimData = getSampleClaimDataBuilder().get().build();
         commonOperations.submitPrePaymentClaim(claimData.getExternalId().toString(), user.getAuthorisation());
 
-        return submitClaim(claimData)
+        return submitClaimWithClaimData(claimData)
             .then()
             .statusCode(HttpStatus.OK.value())
             .and()
             .extract().body().as(Claim.class);
     }
 
-    private Response submitClaim(ClaimData claimData) {
+    private Response submitClaimWithClaimData(ClaimData claimData) {
```

Comment:
revert change, doesn't improve readability, 
it read before as `submitClaim and provide claim data`